README-ta.md

```diff
@@ -0,0 +1,71 @@
+# குபேர்ண்ட்ஸ் ஆவணங்கள்
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+வரவேற்பு! இக்களஞ்சிம்தான் [குபேர்ண்ட்ஸ் வலைத்தளம் மற்றும் ஆவணங்களின்](https://kubernetes.io/) இருப்பிடமாக உள்ளது! நீங்கள் பங்களிக்க விரும்புவதில் நாங்கள் மகிழ்ச்சியடைகிறோம்!
+
+## ஆவணங்களுடன் பங்களிக்க
+
+நீங்கள் இந்த களஞ்சியத்தை போர்க் செய்ய மேல் வலது உள்ள பொத்தானை அழுத்தி உங்களுக்கான பிரதியை  உங்களது கிட்ஹப் கணக்கில் பெற்று கொள்ளலாம். இந்த பிரதியை போர்க் என்று கூறலாம். எந்த மற்றம் வேண்டுமானாலும் போர்க்கில் செய்து கொள்ளலாம்,  உங்கள் மாற்றத்தை எங்களுக்கு அனுப்ப போர்க்கில் சென்று புள் ரெகுஸ்ட் உருவாக்கி எங்களிடம் தெரிவித்து கொள்ளலாம்.
+
+ஒருமுறை உங்களது புள் ரெகுஸ்ட்டை உருவாக்கிய பிறகு, குபேர்ண்ட்ஸ் திறனாய்வாளர் தெளிவான மற்றும் செயல்படக்கூடிய கருத்துக்கு பொறுப்புஏற்றுக்கொள்வர். புள் ரெகுஸ்ட் உரிமையாளராக **உங்களுக்குத்தான் குபேர்ண்ட்ஸ் திறனாய்வாளர் வழங்கும் பிண்ணூட்டத்தை மற்ற பொறுப்பு.** மற்றும் இதை குறித்து கொள்ளுங்கள், அங்கு உங்களுக்கு ஒன்றுக்கு மேற்பட்ட திறனாய்வாளர் இருக்கலாம் இல்லையென்றால் உங்களுக்கு என ஒதுக்கப்பட்ட திறனாய்வாளர் இல்லாமல் வேறு திறனாய்வாளர் திறனாய்வு செய்வர். மேலும், ஒரு சில கட்டங்களில், திறனாய்வாளர் தொழில்நுட்ப திறனாய்வு செய்ய [குபேர்ண்ட்ஸ் தொழில்நுட்ப திறனாய்வாளர்](https://github.com/kubernetes/website/wiki/Tech-reviewers) ஐ அணுகுவர். திறனாய்வாளர் தங்களது சிறந்த பின்னூட்டத்தை தக்க சமையத்தில் வழங்குவர் ஆனால் பதில் நேரம் சூழ்நிலைகளின் அடிப்படையில் மாறுபடும்.
```

Comment:
ஒருமுறை உங்களது [புள் ரெகுஸ்ட்டை => இழுவை கோரிக்கையை] உருவாக்கிய பிறகு, குபேர்ண்ட்ஸ் திறனாய்வாளர் தெளிவான மற்றும் செயல்படக்கூடிய கருத்துக்கு [பொறுப்புஏற்றுக்கொள்வர் => Need space between the two words]. [புள் ரெகுஸ்ட் = இழுவை கோரிக்கை] உரிமையாளராக **உங்களுக்குத்தான் குபேர்ண்ட்ஸ் திறனாய்வாளர் வழங்கும் பிண்ணூட்டத்தை [மற்ற => மாற்ற] பொறுப்பு.** மற்றும் இதை குறித்து கொள்ளுங்கள், அங்கு உங்களுக்கு ஒன்றுக்கு மேற்பட்ட திறனாய்வாளர்[கள்] இருக்கலாம் இல்லையென்றால் உங்களுக்கு என ஒதுக்கப்பட்ட [திறனாய்வாளர் இல்லாமல் => திறனாய்வாளர்களை தவிர] வேறு திறனாய்வாளர் திறனாய்வு செய்வர். மேலும், ஒரு சில கட்டங்களில், திறனாய்வாளர் தொழில்நுட்ப திறனாய்வு செய்ய [குபேர்ண்ட்ஸ் தொழில்நுட்ப திறனாய்வாளர்](https://github.com/kubernetes/website/wiki/Tech-reviewers) ஐ அணுகுவர். திறனாய்வாளர் தங்களது சிறந்த பின்னூட்டத்தை தக்க சமையத்தில் வழங்குவர் ஆனால் [பதில் நேரம் => பதில் அளிக்கும் நேரம்] சூழ்நிலைகளின் அடிப்படையில் மாறுபடும்.

Reply:
I thank the author for his effort. I would also like to help you on this.